### PR TITLE
Added user friendly HTML GUI

### PR DESCRIPTION
**Pull request by shivasheeshyadav**
@thetomja HTML GUI added.
Referencing Issue #1

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -45,6 +45,7 @@ var todoList = {
 
 		// Store data in local storage
 		this.storeData();
+		document.forms['form1'].reset();
```
Comment (thetomja) [inline script.js]: Move this to a separate `resetForm()` function, and call that instead to avoid duplicate code.

File: script.js [hunk 2]
```diff
@@ -56,6 +57,7 @@ var todoList = {
...
```
Comment (thetomja) [inline script.js]: Same as above.

File: script.js [hunk 3]
```diff
@@ -67,14 +69,15 @@ var todoList = {
...
```
Comment (thetomja) [inline script.js]: Ditto.

File: index.html [hunk 4]
```diff
@@ -3,12 +3,41 @@
 <head>
 	<title>Simple Todo List</title>
 	<link rel="stylesheet" type="text/css" href="styles.css">
-	<!-- <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script> -->
+	<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
```
Comment (thetomja) [inline index.html]: If you're not using jQuery, don't load it.

File: script.js [hunk 5]
```diff
@@ -67,14 +69,15 @@ var todoList = {
...
 	},
 
 	setCategory: function(item) {
 		for (i = 1; i < arguments.length; i++) { //Go through each argument
 			for (j in this.items) { //Go through each item in todoList
 				if (this.items[j].val == item) { //If item has a match, set category for it to chosen args
 					if (this.items[j].categories.indexOf(arguments[i]) != -1) {
-						console.log("Selected item already has category: " + arguments[i]);
```
Comment (thetomja) [inline script.js]: Leave the log statements in, so we can use it as both a command line and GUI app.

File: script.js [hunk 6]
```diff
@@ -89,25 +92,33 @@ var todoList = {
 
 		// Store data in local storage
 		this.storeData();
+		document.forms['form1'].reset();
 	},
 
 	showList: function() {
+		document.getElementById("display").innerHTML = "";
 		for (i in this.items) {
-			console.log("- " + this.items[i].val);
```
Comment (thetomja) [inline script.js]: Add the log statements back in.

File: script.js [hunk 7]
```diff
@@ -89,25 +92,33 @@ var todoList = {
...
+			document.getElementById("display").innerHTML += ("- " + this.items[i].val + '<br>');
 		}
 	},
 
 	showListByCategory: function() {
+		document.getElementById("display").innerHTML = "";
 		for (i in this.categories) { //Go through categories
-			console.log('===== ' + this.categories[i].toUpperCase() + ' ====='); //Print category heading-style
```
Comment (thetomja) [inline script.js]: Ditto.

File: script.js [hunk 8]
```diff
@@ -89,25 +92,33 @@ var todoList = {
...
+			document.getElementById("display").innerHTML += ('===== ' + this.categories[i].toUpperCase() + ' =====' + '<br>'); //Print category heading-style
 			for (j in this.items) { //Search items for matching category
 				if (this.items[j].categories.indexOf(this.categories[i]) != -1) {
-					console.log('- ' + this.items[j].val);
```
Comment (thetomja) [inline script.js]: Ditto.

**Comment by shivasheeshyadav** [comment]
@thetomja Made the requested changes.

**Pull request merged**